hazelcast-spring/src/test/resources/com/hazelcast/spring/cache/jCacheClientCacheManager-applicationContext-hazelcast.xml

```diff
@@ -24,7 +24,7 @@
 		http://www.springframework.org/schema/cache
 		http://www.springframework.org/schema/cache/spring-cache.xsd
 		http://www.hazelcast.com/schema/spring
-		http://www.hazelcast.com/schema/spring/hazelcast-spring-3.6.xsd">
+		http://www.hazelcast.com/schema/spring/hazelcast-spring-3.7.xsd">
```

Comment:
I know this is annoying but we have an issue for that. I hope we get it done soon.